.ci/static-checks.sh

```diff
@@ -536,14 +541,77 @@ check_docs()
 			git ls-remote "$url" > /dev/null 2>&1 && continue
```

Comment:
`git ls-remote` is accessing the network, so it could also benefit from using parallel sub-processes like for `curl`

Reply:
Good catch. I think that's a further optimisation that can be done on a follow-on PR though as it will complicate the code yet further.